test/integration/smoke/test_scale_vm.py

```diff
@@ -211,7 +211,6 @@ def test_01_scale_vm(self):
             serviceofferingid=self.small_offering.id,
             mode=self.services["mode"]
         )
```

Comment:
```suggestion
        )
        self.cleanup.append(self.virtual_machine)
```
the line below should not be deleted